### PR TITLE
Blog post Prague 2019 talks announced, CFV, tickets, shirts

### PR DESCRIPTION
**Pull request by thatdocslady**

### CODE REVIEW

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst
Comment (plaindocs) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]: I'd go better than "we hope" :-D

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst
Comment (plaindocs) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]: Twice is maybe too much?

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst
Comment (plaindocs) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]:
```suggestion
We are so grateful to have our sponsors help in bringing these events to life every year. Thanks to the following companies for supporting the Write the Docs community:
```

Just cos when you say sincerely I assume you're not :-p

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst
Comment (plaindocs) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]:
```suggestion
Over the next weeks we'll continue to publish updates about the various events and activities that make Write the Docs events such a unique experience, as well as the presentation schedule, trip planning tips, etc.
```

**Review by plaindocs — COMMENTED** [review]
@thatdocslady it would make life a lot easier if you picked one of the following:

- added the blog post directly in the speaker PR 
- branched from the branch used by the speaker PR and made the PR against that

This way the speaker data is broken till we merge the other PR.

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst
Comment (thatdocslady) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]:
```suggestion
Head over to the `Speakers page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/speakers/>`_ to see the full abstracts. We believe there’s something there for everyone to enjoy, and also something that will broaden your horizons a bit.
```

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst
Comment (thatdocslady) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]:
```suggestion
```

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst
Comment (thatdocslady) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]:
```suggestion
```

**Review by thatdocslady — COMMENTED** [review]
Implemented feedback

**Comment by thatdocslady** [comment]
@plaindocs would you mind pulling in my changes to the main speakers PR? My git-foo stands humbled...

**Comment by plaindocs** [comment]
I looked at changing the PR branch to the speaker PR branch instead of master, but there is a whole host of other changes there as well. I could just cherry-pick these three commits I guess, but probably the easiest is just to merge that speaker PR as soon as we update from Grorg for the last time, and merge this after.

**Comment by thatdocslady** [comment]
No worries, that's what I originally proposed ;)

I'l wait until you merge your PR, rebase from master and test the output, and then if no other feedback then I'll merge mine. Assuming you'll do this tomorrow, I should be online and responsive to GH notifications.

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst
Comment (mxsasha) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst:75]: Is "sport your new threads" a sufficiently well-known expression?

**Review by mxsasha — APPROVED** [review]

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst
Comment (thatdocslady) [reply docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst:75]: I don't think there's a problem with it, do you have strong feelings about it?

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst
Comment (thatdocslady) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]:
```suggestion
Announcing Speakers and Talks, Call for Volunteers, Ticket Update, Shirts on Sale
```

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst
Comment (mxsasha) [reply docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst:75]: No strong feelings. If you think it's fine, let's leave it in.

**Pull request merged**
